tests/lms/models/lis_result_sourcedid_test.py

```diff
@@ -0,0 +1,100 @@
+# import datetime
+
+import pytest
+import sqlalchemy.exc
+
+from lms.models import ApplicationInstance, LISResultSourcedId
+
+
+class TestLISResultSourcedId:
```

Comment:
I kind of took my best whack at tests on this model; not sure if this is going the right direction or not.

Reply:
They look perfect to me